ams/system.py

```
@@ -803,24 +843,59 @@ def to_json(self, outfile: str, overwrite: bool = None):
         """
-        return write_json(self, outfile=outfile, overwrite=overwrite)
+        return write_json(self, outfile=outfile,
+                          skip_empty=skip_empty, overwrite=overwrite,
+                          to_andes=to_andes)
 
-    def to_raw(self, outfile: str, overwrite: bool = None):
+    def to_raw(
+        self,
+        outfile: str,
+        overwrite: Optional[bool] = None,
+        skip_empty: bool = True,
```

Ruff (0.12.2)

Unused method argument: `skip_empty`

(ARG002)